pom.xml

```diff
@@ -52,20 +52,19 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>${workflow.version}</version>
+            <version>2.2.blockapis-SNAPSHOT</version> <!-- allows consuming the new APIs -->
```

Comment:
fwiw, the appropriate way to do that in Maven versioning would be something more like `2.2-blockapis-SNAPSHOT` - see http://stackoverflow.com/a/13007017